package.json

```diff
@@ -5,9 +5,9 @@
     "types": "http/index.d.ts",
     "scripts": {
         "test": "npm run test:unit:all && npm run test:e2e:all",
-        "test:load": "jest --runInBand --detectOpenHandles --forceExit",
+        "test:load": "jest --runInBand",
         "test:load:http": "TEST_SECRET_1=secret_val_1 TEST_SECRET_2=secret_val_2 dapr run --app-id test-suite --app-protocol http --app-port 50001 --dapr-http-port 50000 --components-path ./test/components npm run test:load 'test/load'",
-        "test:e2e": "jest --runInBand --detectOpenHandles --forceExit",
```

Comment:
Curious to know, why are we removing this?

Reply:
`--forceExit` is seen as a force to exit the tests, but they might fail and still succeed then even if the app hangs. `--detectOpenHandles` Will check the open handles and fail the tests then.

They are more meant as a "debug" flag

Reply:
Got it, thanks!